cultural_ads.html

```diff
@@ -0,0 +1,592 @@
+            display: flex;
+            flex-wrap: nowrap;
+            justify-content: center;
+        }
+        #explore-dataset, #explore-categories {
+            margin-top: 20px;
+            display: none;
+        }
+        .example-node {
+            margin-left: 20px;
+        }
+        .country-node {
+            margin-left: 40px;
+        }
+        .image-gallery {
+        display: flex;
+        flex-wrap: wrap;
+        justify-content: center;
+        gap: 10px;
+        margin-top: 20px;
+    }
+    .image-gallery img {
+        max-width: 100%;
+        height: auto;
+        object-fit: contain;
+        max-height: 200px; /* Adjust this value as needed */
+    }
+    .compare-container {
+        border: 1px solid #ddd;
+        border-radius: 8px;
+        padding: 20px;
+        margin: 20px 10px;
+        height: 100%;
+    }
+    .compare-container h3 {
+        margin-bottom: 20px;
+    }
+        .image-gallery .placeholder {
+            margin: 5px;
+            border: 1px solid #ddd;
+            border-radius: 4px;
+            padding: 5px;
+            width: 150px;
+            height: 150px;
+            display: inline-block;
+            background-color: #f0f0f0;
+            text-align: center;
+            line-height: 150px;
+            color: #bbb;
+            font-size: 14px;
+        }
+        .level {
+            margin-bottom: 30px;
+        }
+        /* .compare-container {
+            border: 1px solid #ddd;
+            border-radius: 8px;
+            padding: 20px;
+            margin: 20px 10px;
+        }
+        .compare-container h3 {
+            margin-bottom: 20px;
+        } */
+    </style>
+</head>
+<body>
+    <section class="hero">
+        <div class="hero-body">
+            <div class="container is-max-desktop">
+                <div class="columns is-centered">
+                    <div class="column has-text-centered">
+                        <h1 class="title is-1 publication-title">
+                            <span class="is-size-2"><span class="is-size-1">C</span>ulture<span class="is-size-1">Q</span>uest</span>
+                        </h1>
+                        <h3 class="title is-3 publication-title">Exploring the Depths of Cultural Contexts in Video Ads</h3>
+                        <div class="is-size-5 publication-authors">
+                            <span class="author-block">
+                                <a href="#">Author 1<sup>*</sup></a>,
+                            </span>
+                            <span class="author-block">
+                                <a href="#">Author 2<sup>*</sup></a>,
+                            </span>
+                            <span class="author-block">
+                                <a href="#">Author 3<sup>*</sup></a>,    
+                            </span>
+                            <span class="author-block">
+                                <a href="#">Author 4</a>,
```

Comment:
name Ritika, you, aanisha, and me here.
Put the affiliations as IIITD and Adobe MDSR for respective people.